components/molecules/CollaborationCard/collaboration-card.tsx

```diff
@@ -0,0 +1,34 @@
+import { User } from "@supabase/supabase-js";
+import Avatar from "components/atoms/Avatar/avatar";
+import Button from "components/atoms/Button/button";
+import React from "react";
+
+export type CollaborationRequestObject = {
+  requestor: DbUser | undefined;
+  outreachMessage: string;
+};
+interface CollaborationCardProps extends CollaborationRequestObject {}
+const CollaborationCard = ({ requestor, outreachMessage }: CollaborationCardProps) => {
+  return (
+    <div className="bg-white flex flex-col gap-4 rounded-2xl p-4 border border-light-slate-6 max-w-[41.625rem]">
```

Comment:
Could you please not use `max-w-[41.625]`,  but use `max-w-2xl`  from tailwind is 42 rem